web/src/topic_list.ts

```diff
@@ -152,7 +161,10 @@ export function spinner_li(): ListInfoNode {
     };
 }
 
-export function is_complete_for_stream_id(stream_id: number): boolean {
+export function is_complete_for_stream_id(
```

Comment:
This function probably isn't named well now that it's not in `stream_topic_history.ts`. Maybe `is_sufficient_topic_history_available`?